src/traces/bar/layout_attributes.js

```diff
@@ -51,5 +51,19 @@ module.exports = {
             'Sets the gap (in plot fraction) between bars of',
             'the same location coordinate.'
         ].join(' ')
-    }
+    },
+    barshowtotal: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'style',
```

Comment:
There is no longer `role` needed for attributes like this one.
Please remove the `role: 'style',` line.